test/e2e/distributedtracing/distributedtracing_test.go

```diff
@@ -91,10 +91,11 @@ func TestServerSpan(t *testing.T) {
 			attribute.Int("net.host.port", port),
 			attribute.String("http.method", "POST"),
 			attribute.String("http.scheme", "http"),
-			attribute.String("http.flavor", "1.1"),
+			attribute.String("net.protocol.version", "1.1"),
 			attribute.Int("http.status_code", 200),
-			attribute.String("http.user_agent", "Go-http-client/1.1"),
+			attribute.String("user_agent.original", "Go-http-client/1.1"),
 			attribute.Int("http.wrote_bytes", 3),
+			attribute.String("http.target", "/v0/data"),
```

Comment:
New attribute. Should we test for it, though?

Also new - or at least not asserted before - are: `net.sock.peer.addr` and `net.sock.peer.port`. Should we test for these too? Might be tricky to get the client port as the tests are done today, but we could perhaps check for presence.

Reply:
Per [this](https://opentelemetry.io/blog/2023/http-conventions-declared-stable/#http-server-span-attributes), `http.target` is now `url.path` and `url.query`. 

Makes sense to check for presence of the new attributes.

Reply:
Also not part of this current version bump. Should we hold off on this to not have back-to-back breaking changes? 🤔

Reply:
Also not part of this current version bump. Should we hold off on this to not have back-to-back breaking changes? 🤔

Reply:
Makes sense to hold off then.